lightning/src/ln/channel.rs

```diff
@@ -3824,9 +3827,18 @@ impl<Signer: Sign> Channel<Signer> {
 						return Err(ChannelError::Close("Peer sent a garbage channel_reestablish with secret key not matching the commitment height provided".to_owned()));
 					}
 					if msg.next_remote_commitment_number > INITIAL_COMMITMENT_NUMBER - self.cur_holder_commitment_transaction_number {
-						return Err(ChannelError::CloseDelayBroadcast(
-							"We have fallen behind - we have received proof that if we broadcast remote is going to claim our funds - we can't do any automated broadcasting".to_owned()
-						));
+						macro_rules! log_and_panic {
```

Comment:
Why is a macro needed? Could just use a variable for `$err_msg`.

Reply:
I believe you can format it first with `let err_msg = format!("..", ..)` and then use `"{}", err_msg` in each statement. Or maybe there is a more idiomatic way of using `format_args!`?

Reply:
Nah, just figured we could do it without using a macro. Given it's gonna panic there's not a huge argument for building the string. Fine either way.